components/definitions/welcome.d.ts

```diff
@@ -44,5 +48,6 @@ declare namespace Welcome {
     browserProfiles: Array<BrowserProfile>,
     showSearchCard: boolean,
     showRewardsCard: boolean
+    countryString: string
```

Comment:
Maybe `countryString` is leftover? I think it could be deleted.

Reply:
Oh yes, well spotted.